docs/topics/http/sessions.txt

```diff
@@ -348,6 +348,16 @@ Bundled serializers
         >>> request.session['0']
         'bar'
 
+
+    By default your serializer in Django does not allow non-UTF8 bytes to be stored in session values, note that using non unicode values for ``request.session`` won't work as expected::
```

Comment:
I always search with the error message for solve, so I think mentioning UnicodeDecodeError somewhere in the documentation will be a good idea. :)